packages/web/docs/src/pages/self-hosting/get-started.mdx

```diff
@@ -89,24 +95,162 @@ the environment variables.
 ```bash
-docker-compose -f docker-compose.community.yml up
-# or of you embedded the environment variables
-docker-compose -f docker-compose.with-env.yml up
+docker compose -f docker-compose.community.yml pull
+```
+
+After it's done, you can start the GraphQL Hive services using `docker compose`.
+
+```bash
+docker compose -f docker-compose.community.yml up
+# or of you embedded the environment variables into your docker compose file and called it `docker-compose.with-env.yml`
+docker compose -f docker-compose.with-env.yml up
 ```
 
 Wait until all the services are up and running.
 
 Congratulations 🥳, you just started your own GraphQL Hive instance.
 
+You'll notice the folder named `.hive` at your root directory that has been mounted from docker
+representing the volumes for the different storages used by Hive services like `postgres` and
+`minio`, etc... So yes data are currently local but we'll discuss that further more in the
+`Next Steps` section.
+
+> Note: deleting this directory will end up with a loss of data, as this directory is acting as a volume for storing the data from Hive dependencies: database and storage
+
+## Testing GraphQL Hive
+
 Visit `http://localhost:8080` in your browser and start using GraphQL Hive! The usage reporting
 endpoint is bound to `http://localhost:8081`. The GraphQL API is bound to `http://localhost:8082`.
 The artifacts bucket is bound to `http://localhost:8083`.
 
+Firstly, you can head to `http://localhost:8080` in your browser and start using GraphQL Hive!
+You'll need to Sign Up for an account by inputting your email and password. And once you do you'll
+see your personal organization that has been automatically created for you by default.
+
+### Creating first project and target
+
+Now that we have GraphQL Hive setup locally, signed up and logged into our dashboard. We can start
+creating our first project which we're gonna push graphql schemas to.
+
+So you're gonna click on the `Create a Project` button on the top right, give it a name and choose
+your project type. You're gonna see all sorts of types `Single` which is the most common and
+there're also `Federation` and `Stitching` but what's also nice is that you can define your own
+custom endpoints for schema validation and schema building. I'm gonna go with `Single` but you can
+choose what best suits your needs. Then click `Create Project`, you're gonna be redirected to your
+project page where you're gonna see 3 listed targets created for you by default:
+
+- `development`
+- `production`
+- `staging`
+
+We're gonna use the `development` target in this guide and as soon as you click it you'll see this
+message:
+
+> Hive is waiting for your first schema.
+>
+> You can publish a schema with Hive CLI and Hive Client
+
+If you already have a Nodejs project with a graphql schema that's great you can skip over the next
+step, but if not so we're gonna initialize a new package in our directory, you can use your
+preferred package manager to do so:
```

Comment:
```suggestion
Now that we have GraphQL Hive setup locally, signed up and logged into our dashboard. We can start
creating our first project which we're going to push graphql schemas to.

So you're going to click on the `Create a Project` button on the top right, give it a name and choose
your project type. You're going to see all sorts of types `Single` which is the most common and
there're also `Federation` and `Stitching` but what's also nice is that you can define your own
custom endpoints for schema validation and schema building. I'm going to go with `Single` but you can
choose what best suits your needs. Then click `Create Project`, you're going to be redirected to your
project page where you're going to see 3 listed targets created for you by default:

- `development`
- `production`
- `staging`

We're going to use the `development` target in this guide and as soon as you click it you'll see this
message:

> Hive is waiting for your first schema.
>
> You can publish a schema with Hive CLI and Hive Client

If you already have a Nodejs project with a graphql schema that's great you can skip over the next
step, but if not so we're going to initialize a new package in our directory, you can use your
preferred package manager to do so:
```

Reply:
basically all `gonna` into `going to` :) Just to keep it in the same style as the rest of the documentation

Reply:
lol